fedimint-core/src/db/mod.rs

```diff
@@ -351,8 +351,16 @@ pub trait IDatabaseTransaction<'a>: 'a + MaybeSend {
 
     async fn raw_remove_entry(&mut self, key: &[u8]) -> Result<Option<Vec<u8>>>;
 
+    /// Returns an stream of key-value pairs with keys that start with
+    /// `key_prefix`. No particular ordering is guaranteed.
```

Comment:
Do we still need this one? Isn't every sane db going to return in order anyway? Hmmm... Maybe not ... if it was like an SQL doing collscan without a key ... no ... but then we always give a "prefix" *of the key*, right? So I can't imagine this not being in order ...